python/tests/spark/sql/codegen/test_fs.py

```diff
@@ -33,37 +34,36 @@
 
 
 @pytest.fixture(scope="module")
-def resnet_spec():
+def resnet_spec(tmp_path_factory):
     # Can not use default pytest fixture `tmp_dir` or `tmp_path` because
     # they do not work with module scoped fixture.
-    with tempfile.TemporaryDirectory() as tmp_dir:
-        tmp_path = Path(tmp_dir)
-        # Prepare model
-        resnet = torchvision.models.detection.fasterrcnn_resnet50_fpn(
-            pretrained=True,
-            progress=False,
-        )
-        model_uri = tmp_path / "resnet.pth"
-        torch.save(resnet, model_uri)
+    tmp_path = tmp_path_factory.mktemp(str(uuid.uuid4()))
```

Comment:
just curious, is this tmp_path being tracked and auto-deleted by `tmp_path_factory`?

Reply:
yeah, it's session scoped